class.jetpack-gutenberg.php

```diff
@@ -403,7 +392,7 @@ public static function get_availability() {
 	 * @return bool
 	 */
 	public static function is_gutenberg_available() {
-		return function_exists( 'register_block_type' );
```

Comment:
If this function is generally pointless now, should we add a deprecated function notice here or is there a future use case for it?